ember-cli-build.js

```diff
@@ -5,6 +5,9 @@ var EmberAddon = require('ember-cli/lib/broccoli/ember-addon');
 module.exports = function(defaults) {
   var app = new EmberAddon(defaults, {
     // Add options here
+    babel: {
+      includePolyfill: true
```

Comment:
The problem is that if WeakMap is undefined, `lodash-es` is trying to export default undefined value, and it seems that during transpilation we getting an object with default property that contains some junk instead of undefined...